### DISCUSSION TITLE
possible to export/import from sortly?

### DISCUSSION THREAD

**Discussion by balajeek** [danricho/where_the#17] [Q&A] [answered]
I use sortly iphone app for a very long time, its on the cloud which i was looking to get rid off and self host myself.
any possible way to move data from sortly to where_the.  I have lot of items and pictures.
thanks in advance.

**Comment by danricho** [comment, marked as answer]
I’ve never used sortly, however *Where The ?!?* is very new and not well tested with many items yet.

Additionally, *Where The ?!?* does **NOT** support images at this stage.

**Comment by balajeek** [comment]
Thanks for your help in setting up the docker container, now that i have the app running, I tried to add a location and items and i am not sure if i understand it correctly!

In reality, let say i have a box with multiple toys, and that box is kept in the Attic in a shelf-1 on top rack.

What i expect is to create a location "Attic", then within the location "Area/Rack" may be, then name of the box "Toys Box-1", then all the items contained in that box.

Also i did not find option to upload or store images?

**Reply by danricho** [reply]
Unfortunately I didn’t create this app to have nested storage options. By design it is currently just locations with stored items… there is a small amount of meta data that can be added to the location (eg. location: attic).

Also, as I mentioned previously, storing images is not a feature at this stage. I’m still considering whether it’s something that should be added and what the images would be stored against (location or item).

I’m reluctant for either as there are alternative apps with those features and from the beginning this app was meant to be simple and remain simple. It is not designed to replace Sortly (I’ve not used it).

I’ll add a new discussion to capture the image storing idea, and maybe one day an efficient, simple solution may be implemented.

**Reply by balajeek** [reply]
Oh i was under impression that it supports images according to your post earlier (see 2nd one) where you said "does support images"
I guess may be its a typo.

**Reply by danricho** [reply]
Apologies. That is a typo. I will fix that.
